new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiConfigMapOverride.java

```diff
@@ -0,0 +1,669 @@
+    String dsModelFile = MODEL_DIR + "/model.jdbc.yaml";
+    Map<String, String> data = new HashMap<>();
+    String configMapName = "dsconfigmap";
+    String cmData = null;
+    StringBuffer patchStr  = null;
+
+    cmData  = assertDoesNotThrow(() -> Files.readString(Paths.get(dsModelFile)),
+        String.format("readString operation failed for %s", dsModelFile));
+    assertNotNull(cmData, String.format("readString() operation failed while creating ConfigMap %s", configMapName));
+    data.put("model.jdbc.yaml",cmData);
+
+    V1ObjectMeta meta = new V1ObjectMetaBuilder()
+        .withLabels(labels)
+        .withName(configMapName)
+        .withNamespace(domainNamespace)
+        .build();
+    V1ConfigMap configMap = new V1ConfigMapBuilder()
+        .withData(data)
+        .withMetadata(meta)
+        .build();
+
+    boolean cmCreated = assertDoesNotThrow(() -> createConfigMap(configMap),
+        String.format("createConfigMap failed for %s", configMapName));
+    assertTrue(cmCreated, String.format("createConfigMap failed while creating ConfigMap %s", configMapName));
+
+    patchStr  = new StringBuffer("[{");
+    patchStr.append("\"op\": \"replace\",")
+            .append(" \"path\": \"/spec/configuration/model/configMap\",")
+            .append(" \"value\":  \"" + configMapName + "\"")
+            .append(" }]");
+    logger.log(Level.INFO, "ConfigMap Patch String: {0}", patchStr);
+
+    patch = new V1Patch(new String(patchStr));
+    boolean cmPatched = assertDoesNotThrow(() -> 
+        patchDomainCustomResource(domainUid,domainNamespace,patch,"application/json-patch+json"),
+        "patchDomainCustomResource(configMap)  failed ");
+    assertTrue(cmPatched, "patchDomainCustomResource(configMap) failed");
+
+    patchStr  = new StringBuffer("[{");
+    patchStr.append(" \"op\": \"replace\",")
+            .append(" \"path\": \"/spec/restartVersion\",")
+            .append(" \"value\": \"2\"")
+            .append(" }]");
+    logger.log(Level.INFO, "RestartVersion Patch String: {0}", patchStr);
+
+    patch = new V1Patch(new String(patchStr));
+    boolean rvPatched = assertDoesNotThrow(() -> 
+        patchDomainCustomResource(domainUid,domainNamespace,patch,"application/json-patch+json"),
+        "patchDomainCustomResource(restartVersion)  failed ");
+    assertTrue(rvPatched, "patchDomainCustomResource(restartVersion) failed");
+    logger.info("Snooze for 60 secs for Introspector to kick off");
+    try { 
+      Thread.sleep(60000);
+    } catch (java.lang.InterruptedException ie) {
+      logger.info("Got InterruptedException during Thread.sleep");
+      fail("Got InterruptedException during Thread.sleep");
+    }
+    // Verify a rolling restart is triggered in a sequential fashion 
+    // admin-server --> managed-server1 --> managed-server2 
+    logger.info("Check admin server pod {0} to be restarted in namespace {1}",
+        adminServerPodName, domainNamespace);
+    checkPodDeleted(adminServerPodName);
```

Comment:
just check the uptime, as per ryan and dongbo's comments in standup

Reply:
yes. In my latest commit, I added a new action method to get PodCreationTime.  Now the test compares the timestamp before and after rolling re-start to make sure the pod has been re-started by introspector